build.gradle

```diff
@@ -0,0 +1,25 @@
+plugins {
+    // Apply the java-library plugin for API and implementation separation.
+    id 'java-library'
+}
+
+java {
+    toolchain {
+        languageVersion = JavaLanguageVersion.of(17)
```

Comment:
I think this should be 11, since scanner is not on 17

Reply:
good catch, I'll fix it